app/indexers/curation_concerns/file_set_indexer.rb

```diff
@@ -22,6 +22,12 @@ def generate_solr_document
         # between files on disk (in fcrepo.binary-store-path) and objects
         # in the repository.
         solr_doc[Solrizer.solr_name('digest', :symbol)] = digest_from_content
+        solr_doc[Solrizer.solr_name('page_count')] = object.page_count
+        solr_doc[Solrizer.solr_name('file_title')] = object.file_title
+        solr_doc[Solrizer.solr_name('duration')] = object.duration
+        solr_doc[Solrizer.solr_name('sample_rate')] = object.sample_rate
+        # solr_doc[Solrizer.solr_name('last_modified')] = object.last_modified
```

Comment:
I was thinking last_modified should be something we could index, but it seems to not be in Hydra::PCDM::File.  I'm not 100% sure if we should just ignore, add a ticket into Hydra::Works, or if there is another field in Hydra::Works that is last_modified